### DISCUSSION TITLE
Product Board of Windows App SDK is not updated

### DISCUSSION THREAD

**Discussion by Naveen61097** [microsoft/WindowsAppSDK#3356] [Q&A]
The product Board of Windows App SDK is not updated since the v1.0 Release. Looks like the product board product is cool and more clear about your timeline and flexible to submit our ideas too. As this is not updated, a question is getting raised. "Are you really care about the proposal ideas if we submit there?"

**Comment by ghost1372** [comment]
Product board is not used anymore because one of wasdk member leaved his job and no one access product board.

**Reply by JanRajnoha** [reply]
Team should come with some alternative. I miss option to see, what's going on

**Comment by Panda-Sharp** [comment]
Thye should simply move to GH or developercommunity.visualstudio.com or feedbackportal.microsoft.com, I see no need to use one more (external) tool.

P.s.
Also it's really weird that one member setup product board with his own account, and no one else has/can have access

**Reply by bogdan-patraucean** [reply]
Very unprofessional indeed. I would not expect such mistakes from a company that big, but it seems like people are people, no matter the company, everyone makes mistakes.

**Reply by JanRajnoha** [reply]
From my POV, problem is mainly with no tool. If they get locked from one (productboard), they should come with another or recreate some other account on productboard, but don't let devs without option to see, what is planned

**Comment by Naveen61097** [comment]
ProductBoard tool is very convenient one to check on the status of all features from our end. I request wasdk team to recreate another one for the same and start tracking on it.

**Reply by Panda-Sharp** [reply]
Ther are many other tools, like GH Milestones and Projects, just to mention the first 2 that comes to my mind, no need to go with 3rd party tools and overcomplicate the maintenance, or risk another issue like this.
